featuretools/selection/selection.py

```diff
@@ -169,10 +170,18 @@ def remove_highly_correlated_features(
         more_complex_name = columns_to_check[i]
         more_complex_col = fm_to_check[more_complex_name]
 
+        # Convert boolean column to be float64
+        if pd.api.types.is_bool_dtype(more_complex_col):
+            more_complex_col = more_complex_col.astype("float64")
+
         for j in range(i - 1, -1, -1):
             less_complex_name = columns_to_check[j]
             less_complex_col = fm_to_check[less_complex_name]
 
+            # Convert boolean column to be float64
+            if pd.api.types.is_bool_dtype(less_complex_col):
+                less_complex_col = less_complex_col.astype("float64")
+
             if abs(more_complex_col.corr(less_complex_col)) >= pct_corr_threshold:
```

Comment:
Noob question but is it a normal case for corr to be `nan`?

Reply:
It's possible for the column type to LatLong or a logical type where you cannot calculate the correlation value.

Reply:
Makes sense. Also, I didn't realize that the feature matrix had columns of all zeros (like `bools = diff_ints`), which makes correlation being `nan` make perfect sense.